runtime/jvmti/jvmtiHelpers.c

```diff
@@ -1929,3 +1929,20 @@ genericWalkStackFramesHelper(J9VMThread *currentThread, J9VMThread *targetThread
 
 	return rc;
 }
+
+#if JAVA_SPEC_VERSION >= 19
```

Comment:
This could use some explanation about the relationship between targetThread and the thread object.

Reply:
Updated the function description in jvmti_internal.h.